src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFAMainPage.java

```diff
@@ -0,0 +1,80 @@
+package org.fundacionjala.sfdc.pageobjects.accounts;
+
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.fundacionjala.sfdc.util.CommonActions;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * SFAMainPage.java
+ * Class that represents the main page of accounts section.
+ */
+public class SFAMainPage extends SalesForceConnection{
+    private WebDriverWait wait;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "wait" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)